src/test/java/org/broadinstitute/consent/http/db/DataAccessRequestDAOTest.java

```diff
@@ -41,6 +42,20 @@
 @ExtendWith(MockitoExtension.class)
 class DataAccessRequestDAOTest extends DAOTestHelper {
 
+  private static DataAccessRequestData createDataAccessRequestData(String darCode) {
+    DataAccessRequestData data = new DataAccessRequestData();
+    data.setProjectTitle("Project Title: " + RandomStringUtils.random(50, true, false));
+    data.setDarCode(darCode);
```

Comment:
It would be nice to get away from this deprecated method. Commenting it out, tests pass for me locally.